src/main/java/uk/gov/hmcts/reform/sandl/snlevents/controllers/UserTransactionController.java

```diff
@@ -26,13 +27,15 @@ public UserTransaction getUserTransactionById(@PathVariable("id") UUID id) {
 
     @PostMapping(path = "/{id}/commit", consumes = MediaType.APPLICATION_JSON_VALUE,
         produces = MediaType.APPLICATION_JSON_VALUE)
-    public UserTransaction commit(@PathVariable("id") UUID id) {
-        return userTransactionService.commit(id);
+    public UserTransactionActionResponse commit(@PathVariable("id") UUID id) {
+        boolean succeeded = userTransactionService.commit(id);
+        return new UserTransactionActionResponse(id, "commit", succeeded);
```

Comment:
that "commit" begs to be enumerated doesn't it?

Reply:
removing this would go quite well with one of your previous comments to just add a new UserTransactionStatus ... have to think about it

Reply:
great